bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/DbLedgerStorage.java

```diff
@@ -197,7 +208,9 @@ public void close() {
 
     private LedgerMetadataIndex ledgerIndex;
     private EntryLocationIndex entryLocationIndex;
-    private LoadingCache<Long, TransientLedgerInfo> transientLedgerInfoCache;
+
+    private static final long LEDGER_INFO_CACHING_TIME_MINUTES = 10;
```

Comment:
Can we use `conf.getFileInfoMaxIdleTime` so it is configurable?

Reply:
The default for `getFileInfoMaxIdleTime` and I think we should have some time based eviction from this cache by default. The bookie might be storing many ledgers, though only a fraction of them might be actually be "active".